cmake/Macros.cmake

```diff
@@ -281,11 +300,21 @@ macro(sfml_add_example target)
         add_executable(${target} ${target_input})
     endif()
 
+    # enable C++17 support
+    target_compile_features(${target} PUBLIC cxx_std_17)
```

Comment:
Why add this? It's redundant since this requirement is already picked up when these targets depend on `SFML::` targets. I removed this a PR back in like February.

Reply:
Will try removing, might be unnecessary.